ONSdigital/ssdc-rm-dockerfiles
# 617 nginx maintenance page

# Motivation and Context
We need a way to host a static maintenance page

# What has changed
added a generic static html page that can be used for a maintenance page when needed (nginx can be re-configured to point it it)

# How to test?
- not much to test here, can build the image and check the file has been mounted correctly
